volumes/volume.go

```diff
@@ -113,17 +112,18 @@ func (v *Volume) ToDisk() error {
 }
 
 func (v *Volume) toDisk() error {
-	data, err := json.Marshal(v)
+	jsonPath, err := v.jsonPath()
 	if err != nil {
 		return err
 	}
-
-	pth, err := v.jsonPath()
+	f, err := os.Create(jsonPath)
```

Comment:
This should really also be at least `0644`, even though it wasn't set to this before.